### DISCUSSION TITLE
dpgen runs and ends normally, but the 02.fp file is empty.

### DISCUSSION THREAD

**Discussion by ziqi-hu** [deepmodeling/dpgen#915] [Q&A] [answered]
I use plumed in the model_devi process, the result is normal, there are trajectories generated, but fp_task=0. 02.fp file is empty
I don't know why.

## model_devi and fp section of param.json
"model_devi_engine": "lammps",
    "model_devi_dt": 0.002,
    "model_devi_skip": 0,
    "model_devi_f_trust_lo": 0.02,
    "model_devi_f_trust_hi": 1.95,
    "model_devi_clean_traj": false,
    "model_devi_plumed"    :  true,
    "model_devi_jobs": [
        {   "sys_idx": [0],"traj_freq": 10,"_idx": "00",
            "template":{ "lmp": "data/md_sys/input.lammps", "plm": "data/md_sys/input.plumed" },
            "rev_mat":{ "lmp": {"V_NSTEPS": [2000], "V_TEMP": [300], "V_PRES": [1]},
                        "plm": {"V_TEMP":  [300], "V_STRIDE": [10]}
                      }
        },
        {   "sys_idx": [0],"traj_freq": 10,"_idx": "01",
            "template":{ "lmp": "data/md_sys/input.lammps", "plm": "data/md_sys/input.plumed" },
            "rev_mat":{ "lmp": {"V_NSTEPS": [5000], "V_TEMP": [300], "V_PRES": [1]},
                        "plm": {"V_TEMP":  [300], "V_STRIDE": [10]}
                      }
        }
    ],
"use_clusters": true,
    "cluster_cutoff": 5.0,
    "cluster_minify": true,
    "use_relative": true,
    "epsilon": 1.0,
    "ratio_failed": 0.20,
    "fp_style": "gaussian",
    "shuffle_poscar": false,
    "fp_task_max": 20,
    "fp_task_min": 5,
    "fp_pp_path": "./",
    "fp_pp_files": [],
    "fp_params":{
            "keywords": "force B3LYP 6-31g(d,p) nosymm",
            "nproc":2 ,
            "multiplicity": "auto"
    }

## dpgen.log
2022-08-24 14:46:41,362 - INFO : start running
2022-08-24 14:46:41,372 - INFO : =============================iter.000000==============================
2022-08-24 14:46:41,372 - INFO : -------------------------iter.000000 task 00--------------------------
2022-08-24 14:46:47,114 - INFO : -------------------------iter.000000 task 01--------------------------
2022-08-24 14:58:50,664 - INFO : -------------------------iter.000000 task 02--------------------------
2022-08-24 14:58:50,682 - INFO : -------------------------iter.000000 task 03--------------------------
2022-08-24 14:58:50,839 - INFO : -------------------------iter.000000 task 04--------------------------
2022-08-24 15:07:01,864 - INFO : -------------------------iter.000000 task 05--------------------------
2022-08-24 15:07:01,868 - INFO : -------------------------iter.000000 task 06--------------------------
2022-08-24 15:07:01,890 - INFO : system 000 has no fp task, maybe the model devi is nan %
2022-08-24 15:07:01,893 - INFO : -------------------------iter.000000 task 07--------------------------
2022-08-24 15:07:01,900 - INFO : -------------------------iter.000000 task 08--------------------------
2022-08-24 15:07:01,905 - INFO : =============================iter.000001==============================
2022-08-24 15:07:01,905 - INFO : -------------------------iter.000001 task 00--------------------------
2022-08-24 15:07:01,953 - INFO : -------------------------iter.000001 task 01--------------------------
2022-08-24 15:07:01,957 - INFO : -------------------------iter.000001 task 02--------------------------
2022-08-24 15:07:01,961 - INFO : -------------------------iter.000001 task 03--------------------------
2022-08-24 15:07:02,128 - INFO : -------------------------iter.000001 task 04--------------------------
2022-08-24 15:19:47,658 - INFO : -------------------------iter.000001 task 05--------------------------
2022-08-24 15:19:47,661 - INFO : -------------------------iter.000001 task 06--------------------------
2022-08-24 15:19:47,691 - INFO : system 000 has no fp task, maybe the model devi is nan %
2022-08-24 15:19:47,694 - INFO : -------------------------iter.000001 task 07--------------------------
2022-08-24 15:19:47,701 - INFO : -------------------------iter.000001 task 08--------------------------
2022-08-24 15:19:47,706 - INFO : =============================iter.000002==============================
2022-08-24 15:19:47,706 - INFO : -------------------------iter.000002 task 00--------------------------
2022-08-24 15:19:47,748 - INFO : -------------------------iter.000002 task 01--------------------------
2022-08-24 15:19:47,752 - INFO : -------------------------iter.000002 task 02--------------------------
2022-08-24 15:19:47,755 - INFO : -------------------------iter.000002 task 03--------------------------
2022-08-24 15:19:47,755 - INFO : finished

## traj file
![image](https://user-images.githubusercontent.com/51811006/186386983-0581a5d7-3c09-4514-af5d-8a59c4927c23.png)

**Comment by HuangJiameng** [comment]
Could you provide `model_devi.out`

**Reply by ziqi-hu** [reply]
I don't know why I can't upload the file, I'll paste it below

## model_devi.out
     step         max_devi_v         min_devi_v         avg_devi_v         max_devi_f         min_devi_f         avg_devi_f
           0       1.034433e-02       1.213763e-04       5.816939e-03       9.875800e-01       7.292204e-08       4.278867e-02
          10       1.049175e-02       1.787706e-04       5.891580e-03       9.510761e-01       2.113666e-03       5.402865e-02
          20       1.065857e-02       9.002566e-05       5.768381e-03       9.460406e-01       2.616153e-03       5.494541e-02
          30       1.052009e-02       8.799372e-05       5.966218e-03       7.968282e-01       3.046320e-03       4.843700e-02
          40       1.034181e-02       1.549376e-04       5.848790e-03       6.865473e-01       4.292376e-03       5.078942e-02
          50       1.055898e-02       7.042537e-05       5.840343e-03       7.452703e-01       2.071104e-03       5.521203e-02
          60       1.037664e-02       6.688626e-05       5.849220e-03       1.143003e+00       4.550543e-03       5.355236e-02
          70       1.025718e-02       1.729234e-04       5.506635e-03       1.587381e+00       2.529189e-03       5.003922e-02
          80       1.080963e-02       3.186384e-04       5.834504e-03       1.782020e+00       4.521251e-03       5.326084e-02
          90       9.880893e-03       2.811816e-04       5.315351e-03       1.994036e+00       3.549005e-03       5.319787e-02
         100       9.716628e-03       4.088746e-04       5.163296e-03       2.009160e+00       4.182016e-03       5.258371e-02
         110       1.006893e-02       4.291005e-04       5.470250e-03       1.653445e+00       3.955235e-03       5.068637e-02
         120       9.200485e-03       3.117345e-04       4.830014e-03       1.904915e+00       2.490564e-03       4.913575e-02
         130       9.854152e-03       2.671309e-04       4.967075e-03       1.936057e+00       4.797162e-03       5.368091e-02
         140       9.902669e-03       2.159049e-04       5.051537e-03       1.622837e+00       1.775879e-03       4.979838e-02
         150       9.595377e-03       1.362567e-04       4.855513e-03       1.471277e+00       1.541467e-03       4.757347e-02
         160       9.348138e-03       2.380805e-04       4.905463e-03       1.647907e+00       1.051180e-03       4.888214e-02
         170       9.052350e-03       2.992002e-04       4.861040e-03       1.667305e+00       2.012171e-03       5.173809e-02
         180       9.169998e-03       3.052610e-04       4.933315e-03       1.625976e+00       3.080515e-03       4.950089e-02
         190       9.040424e-03       3.151474e-04       4.925129e-03       1.190193e+00       1.711170e-03       4.213857e-02
         200       9.391906e-03       2.675464e-04       5.046597e-03       1.156603e+00       2.283072e-03       5.357404e-02
         210       8.932497e-03       3.090342e-04       4.813906e-03       1.263001e+00       2.903592e-03       5.070920e-02
         220       9.335741e-03       3.680851e-04       5.194564e-03       1.404585e+00       4.325601e-03       4.320262e-02
         230       9.050348e-03       4.515526e-04       4.917010e-03       1.359024e+00       4.065149e-03       4.864152e-02
         240       8.929829e-03       4.049956e-04       4.910710e-03       1.057022e+00       2.310612e-03       5.101503e-02
         250       9.276217e-03       4.219406e-04       5.094688e-03       9.801592e-01       5.388537e-03       4.656196e-02
         260       8.463622e-03       4.340249e-04       4.593358e-03       1.203177e+00       1.506063e-03       4.361248e-02
         270       9.207945e-03       4.313196e-04       5.090066e-03       1.185012e+00       4.266405e-03       4.899990e-02
         280       8.628017e-03       5.826841e-04       4.674933e-03       1.297198e+00       3.789171e-03       4.629701e-02
         290       8.221548e-03       6.115231e-04       4.629537e-03       1.359994e+00       3.332249e-03       4.459687e-02
         300       8.869038e-03       4.662317e-04       5.009603e-03       1.227111e+00       2.142335e-03       4.482895e-02
         310       8.147555e-03       5.200704e-04       4.480364e-03       9.949549e-01       1.389591e-03       4.528044e-02
         320       8.258717e-03       5.001136e-04       4.611847e-03       7.188075e-01       1.984974e-03       4.841369e-02
         330       8.945027e-03       5.039240e-04       4.819097e-03       1.250337e+00       1.656118e-03       4.437730e-02
         340       8.370251e-03       4.293493e-04       4.488789e-03       8.107664e-01       3.206436e-03       4.525898e-02
         350       8.008771e-03       3.862545e-04       4.470217e-03       1.334398e+00       3.139454e-03       4.915682e-02
         360       8.471386e-03       3.034688e-04       4.607527e-03       1.284344e+00       2.774747e-03       5.016407e-02
         370       8.275253e-03       3.819110e-04       4.544189e-03       1.685700e+00       4.403317e-03       4.749940e-02
         380       7.963902e-03       4.202414e-04       4.362407e-03       1.595552e+00       3.079630e-03       4.567492e-02
         390       8.695174e-03       3.645543e-04       4.576094e-03       1.676239e+00       5.628774e-03       5.437493e-02
         400       7.732320e-03       5.382451e-04       4.245325e-03       1.449366e+00       1.163834e-03       4.834913e-02
         410       8.305769e-03       4.372267e-04       4.608296e-03       1.288884e+00       5.137830e-03       4.185811e-02
         420       8.316148e-03       3.561385e-04       4.561189e-03       9.639874e-01       3.477212e-03       5.039804e-02
         430       7.639220e-03       3.689148e-04       4.311559e-03       1.103987e+00       1.400184e-03       4.906186e-02
         440       8.890142e-03       4.078364e-04       4.864520e-03       1.099334e+00       4.026336e-03       4.274951e-02
         450       8.134198e-03       4.828876e-04       4.411421e-03       7.450430e-01       1.370999e-03       4.342523e-02
         460       8.620775e-03       5.018836e-04       4.862336e-03       6.750839e-01       1.927040e-03       4.952209e-02
         470       8.469392e-03       3.937490e-04       4.669142e-03       9.078906e-01       1.737745e-03       4.371798e-02
         480       8.087202e-03       5.022441e-04       4.641684e-03       1.329860e+00       1.598660e-03       4.417745e-02
         490       8.756364e-03       4.276700e-04       5.013617e-03       1.252680e+00       2.722017e-03       4.700575e-02
         500       8.270398e-03       4.141454e-04       4.567172e-03       1.279624e+00       1.547527e-03       4.627443e-02
         510       8.373991e-03       5.541424e-04       4.738233e-03       1.412279e+00       2.861966e-03       4.906689e-02
         520       8.807844e-03       4.370979e-04       4.857028e-03       1.081938e+00       1.772709e-03       4.362097e-02
         530       8.293148e-03       3.547678e-04       4.553648e-03       1.099024e+00       5.099944e-03       4.877198e-02
         540       7.908658e-03       9.545937e-05       4.496671e-03       1.534367e+00       3.685647e-03       5.030950e-02
         550       8.666185e-03       3.992041e-04       4.877613e-03       1.425380e+00       4.288821e-03       4.787540e-02
         560       8.072013e-03       3.422155e-04       4.550081e-03       1.614523e+00       4.048046e-03       4.599675e-02
         570       8.121289e-03       1.843037e-04       4.593311e-03       1.654036e+00       3.503833e-03       4.645370e-02
         580       8.735670e-03       1.669476e-04       4.853115e-03       1.391738e+00       5.050944e-03       5.034524e-02
         590       8.148416e-03       2.074333e-04       4.577651e-03       1.146474e+00       1.919927e-03       4.273148e-02
         600       8.498855e-03       2.872263e-04       4.881991e-03       8.525181e-01       2.972137e-03       4.087833e-02
         610       8.498910e-03       2.134007e-04       4.809157e-03       7.560522e-01       1.872144e-03       4.879841e-02
         620       7.837255e-03       1.259458e-04       4.509931e-03       8.492016e-01       1.351548e-03       4.494022e-02
         630       8.894512e-03       2.716673e-04       5.024902e-03       1.345721e+00       2.988949e-03       3.984131e-02
         640       8.143856e-03       4.039141e-04       4.540250e-03       1.866639e+00       4.952891e-04       4.608809e-02
         650       8.400340e-03       3.282566e-04       4.689966e-03       2.058436e+00       2.946378e-03       5.041262e-02
         660       8.370395e-03       1.636296e-04       4.728053e-03       2.143810e+00       2.372259e-03       4.398942e-02
         670       7.621665e-03       1.410534e-04       4.385262e-03       1.990504e+00       3.920867e-03       4.578520e-02
         680       8.354774e-03       1.362924e-04       4.719111e-03       1.687670e+00       3.571644e-03       5.070609e-02
         690       7.632673e-03       1.853784e-04       4.324443e-03       1.426746e+00       2.818418e-03       4.592313e-02
         700       7.996985e-03       1.687463e-04       4.508348e-03       8.501402e-01       3.355620e-03       4.674079e-02
         710       8.148787e-03       1.549746e-04       4.598145e-03       6.684948e-01       1.023950e-03       4.260612e-02
         720       8.075515e-03       3.057571e-04       4.510214e-03       6.700009e-01       3.554710e-03       4.694637e-02
         730       7.450632e-03       3.419925e-04       4.241085e-03       6.916200e-01       3.076368e-03       4.572035e-02
         740       8.118804e-03       2.399976e-04       4.652797e-03       7.661834e-01       2.334614e-03       4.189748e-02
         750       7.578858e-03       3.386071e-04       4.280547e-03       1.335918e+00       2.599975e-03       4.419594e-02
         760       7.207851e-03       2.082404e-04       4.085993e-03       9.469242e-01       9.011799e-04       4.505910e-02
         770       8.310403e-03       1.699787e-04       4.636073e-03       1.067746e+00       2.140365e-03       4.718693e-02
         780       7.408013e-03       3.047139e-04       4.137166e-03       1.053266e+00       1.627671e-03       4.121438e-02
         790       7.864025e-03       1.867054e-04       4.505796e-03       9.866402e-01       2.047442e-03       4.479291e-02
         800       7.946181e-03       1.892488e-04       4.471072e-03       9.500396e-01       2.203235e-03       4.934366e-02
         810       7.608562e-03       1.438086e-04       4.313494e-03       7.779810e-01       2.804131e-03       4.421354e-02
         820       8.447278e-03       1.930852e-04       4.679697e-03       6.352826e-01       4.220345e-03       4.090154e-02
         830       8.181392e-03       3.595372e-04       4.529091e-03       5.536217e-01       1.524711e-03       4.744876e-02
         840       7.866298e-03       3.435396e-04       4.494797e-03       7.120590e-01       5.059591e-03       4.742817e-02
         850       8.245502e-03       3.954484e-04       4.691233e-03       7.327728e-01       2.577115e-03       3.929123e-02
         860       7.777388e-03       3.924198e-04       4.506633e-03       9.815445e-01       3.439898e-03       4.405287e-02
         870       8.067449e-03       4.577460e-04       4.632286e-03       1.210266e+00       3.354000e-03       4.780740e-02
         880       7.983849e-03       4.501133e-04       4.528578e-03       1.041307e+00       1.593978e-03       4.116217e-02
         890       8.082049e-03       2.624083e-04       4.633777e-03       1.244388e+00       2.876323e-03       4.347782e-02
         900       8.296206e-03       2.092482e-04       4.616212e-03       1.153512e+00       7.884840e-04       4.309768e-02
         910       7.984170e-03       3.685275e-04       4.532667e-03       8.700031e-01       3.033235e-03       4.507012e-02
         920       8.134263e-03       3.433936e-04       4.507854e-03       8.983385e-01       1.835240e-03       4.455801e-02
         930       8.151432e-03       4.094830e-04       4.635203e-03       8.543042e-01       2.061453e-03       4.079696e-02
         940       8.013701e-03       3.159093e-04       4.551838e-03       1.144702e+00       2.689238e-03       4.614707e-02
         950       7.601202e-03       3.755806e-04       4.254141e-03       1.579580e+00       1.244657e-03       4.615364e-02
         960       8.696607e-03       2.299061e-04       4.843337e-03       1.533562e+00       1.851271e-03       4.478058e-02
         970       7.618506e-03       3.073379e-04       4.269360e-03       1.422501e+00       1.045596e-03       4.174751e-02
         980       8.406222e-03       2.130188e-04       4.549887e-03       1.364008e+00       2.786921e-03       4.715436e-02
         990       8.051796e-03       3.207598e-04       4.596154e-03       1.361766e+00       1.920493e-03       4.685810e-02
        1000       7.625623e-03       2.794875e-04       4.333143e-03       1.013171e+00       2.088489e-03       4.128082e-02
        1010       8.518744e-03       2.825786e-04       4.751576e-03       9.933473e-01       4.362670e-03       4.317020e-02
        1020       7.793558e-03       3.174677e-04       4.431811e-03       7.905623e-01       1.871229e-03       4.590860e-02
        1030       8.248998e-03       3.347632e-04       4.585335e-03       6.732883e-01       3.104978e-03       4.529720e-02
        1040       8.101779e-03       3.032569e-04       4.564334e-03       6.714587e-01       1.336087e-03       3.785825e-02
        1050       7.856582e-03       2.808274e-04       4.519703e-03       8.176226e-01       3.700115e-03       4.617134e-02
        1060       7.953668e-03       3.429634e-04       4.473727e-03       6.887619e-01       2.876711e-03       4.729981e-02
        1070       8.037585e-03       2.863971e-04       4.549728e-03       1.053187e+00       1.570474e-03       4.043109e-02
        1080       7.770654e-03       3.951895e-04       4.462597e-03       7.855539e-01       2.846469e-03       4.386271e-02
        1090       7.999252e-03       3.225647e-04       4.555412e-03       7.733553e-01       1.305300e-03       4.600473e-02
        1100       7.794551e-03       2.044243e-04       4.479431e-03       8.620536e-01       3.257526e-03       4.525997e-02
        1110       7.819846e-03       3.666731e-04       4.430229e-03       6.173163e-01       1.828275e-03       4.376980e-02
        1120       8.011174e-03       2.108384e-04       4.638522e-03       8.903587e-01       2.481271e-03       4.389119e-02
        1130       7.837123e-03       1.574312e-04       4.467445e-03       7.371745e-01       2.132409e-03       4.689540e-02
        1140       7.863146e-03       2.072510e-04       4.351529e-03       7.547952e-01       2.280094e-03       4.603089e-02
        1150       8.649753e-03       2.646005e-04       4.842803e-03       1.287400e+00       4.507106e-03       4.370608e-02
        1160       7.926542e-03       1.979984e-04       4.401882e-03       9.869138e-01       1.400187e-03       4.461452e-02
        1170       7.952935e-03       9.912531e-05       4.446247e-03       1.236318e+00       3.213879e-03       4.882180e-02
        1180       8.539705e-03       9.968776e-05       4.851129e-03       8.722294e-01       2.146179e-03       4.630575e-02
        1190       7.633495e-03       6.744544e-05       4.303234e-03       7.367124e-01       3.406852e-03       4.147524e-02
        1200       8.533755e-03       9.514520e-05       4.802210e-03       8.989522e-01       4.605926e-03       4.779591e-02
        1210       8.101547e-03       1.935210e-04       4.527147e-03       6.227744e-01       1.948139e-03       4.619452e-02
        1220       8.301032e-03       9.494054e-05       4.561587e-03       8.127381e-01       2.041240e-03       4.462018e-02
        1230       8.399718e-03       1.061200e-04       4.720621e-03       8.946251e-01       1.499110e-03       4.112470e-02
        1240       8.131072e-03       1.961283e-04       4.591779e-03       9.930562e-01       2.300730e-03       4.817505e-02
        1250       8.009199e-03       5.725855e-05       4.524312e-03       1.457142e+00       2.288925e-03       4.720051e-02
        1260       8.660535e-03       2.880655e-04       4.745944e-03       1.218265e+00       1.176956e-03       3.987217e-02
        1270       8.393943e-03       1.160280e-04       4.673514e-03       1.291649e+00       2.946101e-03       4.670119e-02
        1280       7.953883e-03       3.136274e-04       4.432848e-03       1.362054e+00       7.954234e-04       4.666932e-02
        1290       8.638262e-03       2.193831e-04       4.883094e-03       9.525235e-01       1.714631e-03       4.506701e-02
        1300       7.722104e-03       8.690290e-05       4.408571e-03       8.207558e-01       1.485258e-03       4.390882e-02
        1310       8.356634e-03       1.195680e-04       4.711853e-03       6.705587e-01       2.362415e-03       4.619251e-02
        1320       8.435755e-03       2.524324e-04       4.596743e-03       6.374694e-01       4.079003e-03       4.685862e-02
        1330       7.799727e-03       3.062728e-04       4.344789e-03       8.202456e-01       1.479815e-03       4.448721e-02
        1340       8.388650e-03       2.882212e-04       4.724530e-03       1.023571e+00       3.919798e-03       4.279995e-02
        1350       7.630786e-03       1.778022e-04       4.363157e-03       1.290937e+00       1.601177e-03       4.569207e-02
        1360       7.801469e-03       2.142050e-04       4.305409e-03       1.139650e+00       2.375951e-03       4.706186e-02
        1370       8.200622e-03       1.834012e-04       4.668029e-03       1.105681e+00       2.189511e-03       4.275185e-02
        1380       7.677191e-03       1.430600e-04       4.256918e-03       1.104033e+00       2.384642e-03       4.238565e-02
        1390       7.758325e-03       3.296667e-04       4.373942e-03       8.878205e-01       3.141544e-03       4.759629e-02
        1400       8.239868e-03       1.915797e-04       4.529967e-03       7.800796e-01       1.821282e-03       4.435583e-02
        1410       7.638178e-03       6.767611e-05       4.348174e-03       7.081316e-01       3.069590e-03       4.224853e-02
        1420       7.888852e-03       3.215358e-04       4.560596e-03       8.257060e-01       1.252700e-03       4.374067e-02
        1430       8.006383e-03       1.451412e-04       4.494056e-03       6.891222e-01       2.857668e-03       4.734031e-02
        1440       7.893960e-03       3.289019e-04       4.481326e-03       6.265361e-01       2.191223e-03       4.457947e-02
        1450       7.989643e-03       3.863477e-04       4.513088e-03       7.831100e-01       2.286033e-03       3.959507e-02
        1460       8.417861e-03       1.619705e-04       4.650362e-03       8.102316e-01       2.135159e-03       4.778273e-02
        1470       7.500320e-03       4.247873e-04       4.250716e-03       7.571189e-01       1.241343e-03       4.481460e-02
        1480       8.475484e-03       4.535513e-04       4.732305e-03       7.923985e-01       2.392415e-03       4.168003e-02
        1490       7.981053e-03       3.590392e-04       4.431672e-03       6.909204e-01       2.598574e-03       4.351963e-02
        1500       7.579112e-03       5.856572e-04       4.379934e-03       7.856744e-01       2.495956e-03       4.556287e-02
        1510       8.159351e-03       4.237476e-04       4.622487e-03       8.544364e-01       2.354947e-03       4.425193e-02
        1520       7.551993e-03       4.906701e-04       4.270835e-03       7.970033e-01       1.460545e-03       4.187787e-02
        1530       8.178841e-03       4.353147e-04       4.674426e-03       7.312891e-01       1.694405e-03       4.434618e-02
        1540       7.659270e-03       3.818609e-04       4.275817e-03       7.603452e-01       1.407245e-03       4.430730e-02
        1550       7.857248e-03       4.480684e-04       4.474215e-03       8.072651e-01       2.018633e-03       4.605750e-02
        1560       8.127982e-03       3.268335e-04       4.619951e-03       8.298100e-01       2.180945e-03       4.084342e-02
        1570       7.888685e-03       2.516770e-04       4.459557e-03       8.338206e-01       2.421093e-03       4.418618e-02
        1580       7.686586e-03       3.278310e-04       4.403431e-03       9.683976e-01       2.096205e-03       4.746238e-02
        1590       8.111030e-03       2.656614e-04       4.606937e-03       8.935018e-01       1.517348e-03       4.204417e-02
        1600       7.861599e-03       2.869573e-04       4.428747e-03       7.597934e-01       2.947288e-03       4.306519e-02
        1610       7.896630e-03       1.543762e-04       4.502381e-03       9.377161e-01       1.381871e-03       4.605326e-02
        1620       8.077906e-03       1.217552e-04       4.608203e-03       9.595660e-01       2.372660e-03       4.620505e-02
        1630       7.814519e-03       2.183591e-04       4.449989e-03       1.204163e+00       2.705136e-03       4.322571e-02
        1640       8.232688e-03       2.636214e-04       4.622778e-03       1.471241e+00       3.973927e-03       4.237297e-02
        1650       8.058906e-03       1.551305e-04       4.570918e-03       1.180742e+00       3.365095e-03       4.770081e-02
        1660       7.744552e-03       1.017886e-04       4.426191e-03       1.212567e+00       1.393287e-03       4.459866e-02
        1670       8.161768e-03       2.195820e-04       4.666162e-03       1.047970e+00       3.839257e-03       4.097656e-02
        1680       7.950993e-03       2.933451e-04       4.538381e-03       1.037004e+00       1.715595e-03       4.619033e-02
        1690       7.571275e-03       1.383989e-04       4.330511e-03       7.296860e-01       2.754664e-03       4.630400e-02
        1700       8.341215e-03       2.243357e-04       4.716459e-03       1.017658e+00       2.733612e-03       4.403862e-02
        1710       7.496680e-03       1.858158e-04       4.283824e-03       9.114720e-01       1.707891e-03       4.269801e-02
        1720       8.247399e-03       3.596810e-04       4.642755e-03       7.242525e-01       1.706761e-03       4.723934e-02
        1730       7.666327e-03       4.597464e-04       4.382785e-03       9.963424e-01       9.127361e-04       4.483903e-02
        1740       7.780319e-03       5.982567e-04       4.395111e-03       8.068144e-01       2.858282e-03       4.481034e-02
        1750       8.080337e-03       3.960156e-04       4.635386e-03       8.294924e-01       1.674444e-03       4.301363e-02
        1760       7.513207e-03       5.548594e-04       4.344120e-03       7.062600e-01       1.626066e-03       4.519211e-02
        1770       7.994535e-03       3.261668e-04       4.479331e-03       8.952356e-01       2.691231e-03       4.720099e-02
        1780       7.934571e-03       3.629317e-04       4.507942e-03       9.326128e-01       2.000665e-03       4.058443e-02
        1790       8.106854e-03       4.415151e-04       4.532403e-03       8.787958e-01       3.278354e-03       4.446662e-02
        1800       7.662669e-03       4.350952e-04       4.288483e-03       8.390902e-01       2.690973e-03       4.634470e-02
        1810       8.304931e-03       4.368672e-04       4.670617e-03       8.724726e-01       1.886559e-03       4.488122e-02
        1820       7.548395e-03       3.402378e-04       4.270737e-03       1.075908e+00       1.676712e-03       4.214245e-02
        1830       8.061934e-03       2.998514e-04       4.510049e-03       1.206157e+00       2.106974e-03       4.495704e-02
        1840       8.338474e-03       3.511596e-04       4.543095e-03       1.238544e+00       2.527799e-03       4.716539e-02
        1850       7.713384e-03       4.237424e-04       4.290327e-03       1.169706e+00       1.328340e-03       4.269398e-02
        1860       8.231793e-03       4.722721e-04       4.640189e-03       1.039274e+00       3.024126e-03       4.201204e-02
        1870       7.968585e-03       4.091422e-04       4.419349e-03       8.788439e-01       9.572040e-04       4.611146e-02
        1880       7.714268e-03       4.191198e-04       4.323302e-03       1.092138e+00       3.944174e-03       4.542509e-02
        1890       8.263217e-03       4.324070e-04       4.604109e-03       1.256114e+00       1.689639e-03       4.155263e-02
        1900       7.926837e-03       3.896289e-04       4.355221e-03       1.248826e+00       1.728565e-03       4.323643e-02
        1910       7.970037e-03       4.295473e-04       4.366387e-03       1.604670e+00       2.819379e-03       4.671663e-02
        1920       8.380587e-03       3.043246e-04       4.523466e-03       1.447727e+00       7.613207e-04       4.359367e-02
        1930       7.928799e-03       2.835203e-04       4.274092e-03       1.265617e+00       2.885990e-03       4.331788e-02
        1940       8.256662e-03       2.786113e-04       4.536500e-03       1.196447e+00       1.805188e-03       4.452000e-02
        1950       7.891189e-03       1.710381e-04       4.283260e-03       9.227811e-01       1.014449e-03       4.480692e-02
        1960       8.270943e-03       1.917626e-04       4.474818e-03       8.643231e-01       1.525273e-03       4.524016e-02
        1970       8.296093e-03       2.639042e-04       4.457627e-03       1.076549e+00       2.436587e-03       3.991120e-02
        1980       8.369620e-03       2.045099e-04       4.467116e-03       1.021585e+00       2.931845e-03       4.529121e-02
        1990       7.995796e-03       1.503426e-04       4.269329e-03       9.556321e-01       2.234519e-03       4.558874e-02
        2000       8.334866e-03       1.992931e-04       4.523548e-03       1.008900e+00       1.760102e-03       4.179721e-02

**Reply by HuangJiameng** [reply]
It seems that task 07 and task 08 usually behave. Only tasks 01-06 get trouble. Are you sure it is a `model_devi.out` for a task in 01-06?

**Reply by ziqi-hu** [reply]
Can I package the file and send it to you to reproduce?

**Reply by HuangJiameng** [reply]
OK

**Comment by ziqi-hu** [comment, marked as answer]
solved!  Need to add "atomic relative 1.0" in lammps input file pair_style.  1.0 for eplsion.

**Reply by goodluck-xyz** [reply]
请问你说的这个添加如何体现在 parameter.json 文件里面

**Reply by flamingoXu** [reply]
I also encounter this problem. Can you provide an example of the lammps input file? thank you

**Comment by sooyaaa233** [comment]
I also have this problem, can you tell me how you specifically solved this problem?
